src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryPipelineSpark.java

```diff
@@ -91,17 +111,45 @@ protected void runTool( final JavaSparkContext ctx ) {
         final List<EvidenceTargetLink> evidenceTargetLinks = assembledEvidenceResults.getEvidenceTargetLinks();
         final PairedStrandedIntervalTree<EvidenceTargetLink> evidenceLinkTree = makeEvidenceLinkTree(evidenceTargetLinks);
 
+        final SVIntervalTree<VariantContext> cnvCalls;
```

Comment:
It looks like these lines are highly similar to those in `DiscoverVariantsFromContigAlignmentsSAMSpark.runTool()`. 
Could you refactor them into a method in `DiscoverVariantsFromContigAlignmentsSAMSpark` and try to have the `runTool()` in `DiscoverVariantsFromContigAlignmentsSAMSpark` thin (easier for the pipeline to call into)?

Reply:
It seemed like we were trying to leave operations involving the spark context, like actually broadcasting variables, within the `runTool` methods of the tools involved, which is why I didn't end up doing as you suggested. If I refactor this way I'll have to pass the spark context object in. What if I just make a static method to do the load and broadcast on `DiscoverVariantsFromContigAlignmentsSAMSpark` and call it from both places?

Reply:
Yep, that would be good.